third_party/rust/idna/v0_2/crate/src/lib.rs

```diff
@@ -0,0 +1,178 @@
+// Copyright 2016 The rust-url developers.
```

Comment:
do we want to keep this copyright notice?

Reply:
Yes, we are including code from the original crate so we need to have both here